Module not found: Can't resolve 'async_hooks'

Upgrading to @vercel/flags 3.1.0 from 2.x results in this error for us:

```
Module not found: Can't resolve 'async_hooks'
  4 |
  5 | // src/lib/tracing.ts
> 6 | import { AsyncLocalStorage } from "async_hooks";
    | ^
  7 | var vercelFlagsTraceSymbol = Symbol.for("@vercel/flags:global-trace");
  8 | function setTracerProvider(tracer) {
  9 |   Reflect.set(globalThis, vercelFlagsTraceSymbol, tracer);
```

Any way to rectify?

--- a/.changeset/hot-clocks-hunt.md
+++ b/.changeset/hot-clocks-hunt.md
@@ -1,0 +1,5 @@
+---
+'@vercel/flags': patch
+---
+
+fix use node: protocol for async_hooks import

--- a/packages/flags/src/lib/tracing.ts
+++ b/packages/flags/src/lib/tracing.ts
@@ -4,8 +4,9 @@ import type {
   TracerProvider,
   AttributeValue,
 } from '@opentelemetry/api';
+import { AsyncLocalStorage } from 'node:async_hooks';
+
 import { name as pkgName, version } from '../../package.json';
-import { AsyncLocalStorage } from 'async_hooks';
 
 // Use a symbol to avoid having global variable that is scoped to this file,
 // as it can lead to issues with cjs and mjs being used at the same time.

